bu-ist/responsive-foundation
# Feature/reduce dependencies fixes

- Replaces grunt-contrib-sass with grunt-sass (which uses node-sass for its implementation)
- Uses full path (include node_modules) to bring in normalize-scss and math sass partials.
- Bugfix on burf-theme.scss (needed to import burf-base.scss).
- Bugfix on icon partial paths (they were incorrectly written using full path from root of repo, as opposed to its directory it was already in).

### Tested in:

1. standalone context where sass was compiling to this repo
2. with bu-prepress which uses create-guten-block and pulls in partials from Foundation
3. with responsive-framework which uses grunt-sass and pulls in partials from Foundation

### Steps to reproduce:

1. Checkout this branch `feature/reduce-dependencies`
2. Checkout responsive-framework's `develop` branch.
3. In Terminal `cd path/to/responsive-framework`
4. Run `npm link path/to/responsive-foundation`
5. Run `grunt` and save a sass partial. Note that it should bomb/error out.

### Steps to test the fix:
1. Checkout this branch `feature/reduce-dependencies-fixes`
2. Checkout responsive-framework's `develop` branch.
3. In Terminal `cd path/to/responsive-framework`
4. Run `npm link path/to/responsive-foundation`
5. Run `grunt` and save a sass partial. Note that it should work (edit: copy/paste mistake 😳 it should *work* not bomb 😉  )
